static/src/javascripts/projects/commercial/modules/dfp/display-lazy-ads.ts

```diff
@@ -0,0 +1,40 @@
+import { dfpEnv } from './dfp-env';
+import { enableLazyLoad } from './lazy-load';
+import { loadAdvert } from './load-advert';
+
+const instantLoadAdvertIds = ['dfp-ad--im'];
+
+const partition = <T>(
+	array: T[],
+	callback: (e: T, i: number, array: T[]) => boolean,
+): [T[], T[]] =>
+	array.reduce(
+		(result: [T[], T[]], element, i) => {
+			callback(element, i, array)
+				? result[0].push(element)
+				: result[1].push(element);
+
+			return result;
+		},
+		[[], []],
+	);
```

Comment:
This is what I was hoping for: a `partition` method.

How 👌 _Magnifique!_

Worth moving up to utils? Or at least exporting it?